src/MagnumPlugins/WavAudioImporter/Test/WavImporterTest.cpp

```diff
@@ -172,6 +172,10 @@ void WavImporterTest::invalidFactChunk() {
 
     CORRADE_COMPARE(importer.format(), Buffer::Format::Mono16);
     CORRADE_COMPARE(importer.frequency(), 22050);
+
+    CORRADE_COMPARE_AS(importer.data().slice(0, 8),
+        Containers::Array<char>::from(-27, -11, -1, -9, 24, -6, 127, -5).slice(0, 8),
```

Comment:
There's some unfortunate limitation in finding the common type in the testsuite that I have to fix, but didn't find out how yet so it's kinda annoying to use, sorry.

In this case, can you do the conversion explicitly, i.e.

```cpp
Containers::ArrayView<const char>{Containers::Array<char>::from(-27, -11, -1, -9, 24, -6, 127, -5)}
```

instead of the `slice()`? Similarly elsewhere.